src/manager.cpp

```diff
@@ -531,29 +542,81 @@ sdbusplus::async::task<>
+        data_sync::async::AsyncCommandExecutor executor(_ctx);
+        auto result = co_await executor.execCmd(notifyCmd);
+
+        switch (result.first)
+        {
+            case 0: // Success
+            {
+                lg2::debug(
+                    "Successfully send notify request[{NOTIFYPATH}] to the sibling BMC "
+                    "for the path[{PATH}]",
+                    "NOTIFYPATH", notifyPath, "PATH", modifiedPath);
+                co_return;
+            }
+
+            case 24: // Vanished source
+            {
+                lg2::error(
+                    "Notify Request[{NOTIFYPATH}] to sibling BMC exited with vanished "
+                    "file error for the path [{PATH}], treating as permanent error.",
+                    "NOTIFYPATH", notifyPath, "PATH", modifiedPath);
+                co_return;
+            }
+
+            default:
+            {
+                if (!isRetryEligible(result.first))
+                {
+                    lg2::error(
+                        "Notify Request[{NOTIFYPATH}] to sibling BMC failed due to permanent error. "
+                        "Modified_path={MOD_PATH}, Error{ERRORCODE} : {ERROR}",
+                        "NOTIFYPATH", notifyPath, "MOD_PATH", modifiedPath,
+                        "ERRORCODE", result.first, "ERROR", result.second);
+                    co_return;
+                }
+            }
+        }
+
+        // NO more retries left
+        if (retryAttempts > cfg._retry->_maxRetryAttempts)
+        {
+            break;
+        }
 
-    auto result = co_await executor.execCmd(notifyCmd);
-    if (result.first != 0)
-    {
-        // TODO : Need to retry if notify command fails.
-        lg2::error(
-            "Failed to send notify request[{NOTIFYPATH}] to the sibling BMC for "
-            "the modified path[{PATH}]; Error[{ERRCODE}] : {ERROR}",
-            "NOTIFYPATH", notifyPath, "PATH", modifiedPath, "ERRCODE",
-            result.first, "ERROR", result.second);
-    }
-    else
-    {
         lg2::debug(
-            "Successfully send notify request[{NOTIFYPATH}] to the sibling BMC "
-            "for the path[{PATH}]",
-            "NOTIFYPATH", notifyPath, "PATH", modifiedPath);
+            "Notify Request[{NOTIFYPATH}] to sibling BMC failed, scheduling retry"
+            "[{RETRY}/{MAX}] after {INTERVAL}s",
+            "NOTIFYPATH", notifyPath, "RETRY", retryAttempts, "MAX",
+            cfg._retry->_maxRetryAttempts, "INTERVAL",
+            cfg._retry->_retryIntervalInSec.count());
+
+        co_await sleep_for(_ctx, std::chrono::seconds(
+                                     cfg._retry->_retryIntervalInSec.count()));
     }
+
+    lg2::error(
+        "Failed to send notify request[{NOTIFYPATH}] to the sibling BMC after "
+        "exhausting[{ATTEMPTS}/{MAX_ATTEMPTS}] retries, Modified path : {MODIFIEDPATH}",
+        "NOTIFYPATH", notifyPath, "ATTEMPTS", cfg._retry->_maxRetryAttempts,
```

Comment:
> cfg._retry->_maxRetryAttempts

Its not correct, why the same value printed twice with different entry id?

Either it should be retryAttempts / cfg._retry->_maxRetryAttempts or just say exhausted max retry [{cfg._retry->_maxRetryAttempts}]